source/object-model/computed-properties-and-aggregate-data.md

```diff
@@ -58,10 +58,9 @@ todosController.get('remaining');
 Note that `@each` only works one level deep. You cannot use nested forms like
 `todos.@each.owner.name` or `todos.@each.owner.@each.name`.
 
-Sometimes, you may only care if the items of an array have been added, removed, or replaced, 
-rather than changes to properties on the items themselves. For example, items may have been 
-added to or removed from the array. In this case, you can create a computed property as follows:
-
+Sometimes you care about changes to the composition of the array (cases 2, 3,
```

Comment:
I don't feel like the callback to the list is beneficial to remove the duplication. If it were right after the list I wouldn't have minded it as much. Also, the working "composition of the array" even with the callbacks is too dense.